icu4c/source/i18n/unicode/uformattedvalue.h

```diff
@@ -22,6 +22,11 @@
 /**
  * All possible field categories in ICU. Every entry in this enum corresponds
  * to another enum that exists in ICU.
+ * 
+ * In the APIs that take a UFieldCategory, an int32_t type is used. ICU will
+ * never exceed 2^16 for its built-in field position values. Field category
```

Comment:
Revise this comment based on TC discussion. ICU gets a larger range.

Reply:
Done.